src/app/components/school-menu/school-menu.tsx

```diff
@@ -7,41 +7,66 @@ import './school-menu.scss';
 const schoolMenuStaticLinks = [
   {
     title: 'About RS School',
-    detailsUrl: '/courses',
+    detailsUrl: '/#about',
     description: 'Free online education',
   },
   {
     title: 'Upcoming courses',
-    detailsUrl: '/courses#upcoming-courses',
+    detailsUrl: '/#upcoming-courses',
     description: 'Schedule your study',
   },
-  {
-    title: 'IT Journey',
-    detailsUrl: '/courses#learning-path',
-    description: 'Plan your developer path',
-  },
   {
     title: 'Mentoring',
-    detailsUrl: '/courses#mentors-wanted',
+    detailsUrl: '/#mentors-wanted',
     description: 'Contribute and study',
   },
 ];
 
+const communityMenuStaticLinks = [
+  {
+    title: 'About',
+    detailsUrl: '/community/#about',
+    description: 'Who we are',
+  },
+  {
+    title: 'Events',
+    detailsUrl: '/community/#events',
+    description: 'Meet us at events',
+  },
+  {
+    title: 'Merch',
+    detailsUrl: '/community/#merch',
+    description: 'Sloths for your daily life',
+  },
+  {
+    title: 'Contribute',
+    detailsUrl: '/community/#contribute',
+    description: 'Assist us and improve yourself',
+  },
+];
+
 interface SchoolMenuProps {
-  heading: 'rs school' | 'all courses';
+  heading: 'rs school' | 'all courses' | 'community';
+  hasTitle?: boolean;
   color?: 'dark' | 'light';
 }
 
-export const SchoolMenu = ({ heading, color = 'light' }: SchoolMenuProps) => {
+export const SchoolMenu = ({ heading, hasTitle = true, color = 'light' }: SchoolMenuProps) => {
   const { data } = useDataByName('courses');
-
   const courses = data as Course[];
+  let schoolListProps: GenericItemProps[] | Course[];
 
-  const schoolListProps = heading.includes('courses') ? courses : schoolMenuStaticLinks;
+  if (heading.includes('courses')) {
+    schoolListProps = courses;
+  } else if (heading.includes('school')) {
+    schoolListProps = schoolMenuStaticLinks;
+  } else {
+    schoolListProps = communityMenuStaticLinks;
+  }
 
   return (
     <div className="school-menu">
-      <h3 className={`heading ${color}`}>{heading}</h3>
+      {hasTitle && <h3 className={`heading ${color}`}>{heading}</h3>}
       <SchoolList list={schoolListProps} color={color} />
```

Comment:
You don't need to change anything, but, God, these component names drive me crazy: SchoolMenu, SchoolList

Reply:
Totally agree